### ISSUE TITLE
0 example value not shown in UI

### ISSUE THREAD

**Issue by SledgeHammer01** [swagger-api/swagger-ui#6352]
I've seen some several old bug reports where it seems like this was fixed a while ago, but still seeing it:

SpringDoc 1.4.5
Swagger-ui 3.32.5

Code looks like:

```
	public Page<Item> getPagedItems(@Parameter(description="The page number to retrieve.", required=true, example="0") @RequestParam int page,
							   		@Parameter(description="The number of items per page.", required=true, example="25") @RequestParam int size) throws MethodArgumentNotValidException {

```

Generated json:

`description":"Gets items by page number & page size.","operationId":"getPagedItems_1","parameters":[{"name":"page","in":"query","description":"The page number to retrieve.","required":true,"schema":{"type":"integer","format":"int32"},"example":0},{"name":"size","in":"query","description":"The number of items per page.","required":true,"schema":{"type":"integer","format":"int32"},"example":25}],
`

Every example value except for 0 works. In the UI, it just renders as blank, so the user has to manually enter the 0. It does show "Example : 0", but the text box is not populated for 0.

**Comment by hkosova** [comment]
Duplicate of #5573.

**Comment by s13o** [comment]
It's not a duplicate, the defect still exists for the `example` attribute. The mentioned defect was about `defaultValue`  where it's resolved but for an `example` attribute the defect still exists

**Comment by hkosova** [comment]
@s13o This issue and #5573 were fixed in Swagger UI v3.35.1. [The fix](https://github.com/swagger-api/swagger-ui/pull/6454) covered both `default` and `example` values of 0.

If you are still having problems with the latest version of Swagger UI, please [open a new issue](https://github.com/swagger-api/swagger-ui/issues/new/choose) and provide a sample OpenAPI definition and other details requested in the issue template.
